en/installation.rst

```diff
@@ -399,6 +399,37 @@ you will need PHP running as a FastCGI instance::
             fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
         }
     }
+    
+One some servers (Like Ubuntu 12.04) the above configuration won't work out of the box, 
```

Comment:
It would be better to reference the latest LTS which is 14.04.  

Also please limit text line lengths to 80 columns.